src/Domain/MigrationStep/s050_DestinationPimInstallation/DestinationPimConfigurationChecker.php

```diff
@@ -21,19 +21,24 @@ class DestinationPimConfigurationChecker
     /** @var DestinationPimSystemRequirementsChecker */
     private $destinationPimSystemRequirementsChecker;
 
-    public function __construct(DestinationPimEditionChecker $destinationPimEditionChecker, DestinationPimSystemRequirementsChecker $destinationPimSystemRequirementsChecker)
+    /** @var DestinationPimVersionChecker */
+    private $destinationPimVersionChecker;
+
+    public function __construct(DestinationPimEditionChecker $destinationPimEditionChecker, DestinationPimSystemRequirementsChecker $destinationPimSystemRequirementsChecker, DestinationPimVersionChecker $destinationPimVersionChecker)
     {
         $this->destinationPimEditionChecker = $destinationPimEditionChecker;
         $this->destinationPimSystemRequirementsChecker = $destinationPimSystemRequirementsChecker;
+        $this->destinationPimVersionChecker = $destinationPimVersionChecker;
     }
 
     public function check(SourcePim $sourcePim, DestinationPim $destinationPim): void
     {
         try {
             $this->destinationPimEditionChecker->check($sourcePim, $destinationPim);
+            $this->destinationPimVersionChecker->check($destinationPim);
             $this->destinationPimSystemRequirementsChecker->check($destinationPim);
         } catch (\Exception $e) {
-            throw new DestinationPimCheckConfigurationException($e->getMessage(), $e->getCode(), $e);
+            throw new DestinationPimCheckConfigurationException($e->getMessage(), $e->getCode());
```

Comment:
Why create specific exception in `DestinationPimTrololo->check()` and catch them here to put it in another exception. Imho it's weird, but now it's done.

Reply:
Yes it's weird, but I didn't want to refactor that in this PR

Reply:
That's already the case, I do not understand neither.

Reply:
It costs almost nothing to delete this `try/catch`